<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwapBackupDlg</class>
 <widget class="QDialog" name="SwapBackupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="control::MwcLabelLarge" name="title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>632</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Backup trade XXXXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label1">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>75</y>
     <width>726</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Please backup the data for the Swap Trade XXXXXXX. </string>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="mainTextLabel">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>120</y>
     <width>736</width>
     <height>96</height>
    </rect>
   </property>
   <property name="text">
    <string>We strongly recommend that you backup the trade data to a separate data storage device, like a USB Drive. In case of storage failure this backup will be the only way to restore the trade and get access to your locked funds.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="backupDataPath">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>255</y>
     <width>596</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200000</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="control::MwcLabelSmall" name="label_3">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>235</y>
     <width>576</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Backup file path</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="selectPath">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>255</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14000</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Select the filename for the backup.</string>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="backupButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14000</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Backup the data</string>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="skipButton">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>340</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14000</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>Skip the backup. I understand that I can loose my funds.</string>
   </property>
   <property name="text">
    <string>Skip</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
